lib/matplotlib/tests/test_axes.py

```diff
@@ -4673,10 +4667,7 @@ def test_pandas_errorbar_indexing():
 
 
 def test_pandas_indexing_hist():
-    try:
-        import pandas as pd
-    except ImportError:
-        pytest.skip("Pandas not installed")
+    pd = pytest.importorskip('pandas')
```

Comment:
It doesn't appear that there is. Creating a `Series` from a list and `DataFrame` a from a dictionary of lists/arrays should work with versions of pandas several years old at this point.